cranelift-codegen/src/verifier/mod.rs

```diff
@@ -1814,7 +1840,8 @@ mod tests {
     use super::{Verifier, VerifierError, VerifierErrors};
     use crate::entity::EntityList;
     use crate::ir::instructions::{InstructionData, Opcode};
-    use crate::ir::Function;
+    use crate::ir::types::*;
+    use crate::ir::{Constant, Function};
```

Comment:
nit: probably some of these are unused now and would generate warnings when compiling tests

Reply:
Ok, let me check before I merge.